src/composables/node/useNodePricing.ts

```diff
@@ -169,6 +169,46 @@ const byteDanceVideoPricingCalculator = (node: LGraphNode): string => {
     : `$${minCost.toFixed(2)}-$${maxCost.toFixed(2)}/Run`
 }
 
+const ltxvPricingCalculator = (node: LGraphNode): string => {
+  const modelWidget = node.widgets?.find(
+    (w) => w.name === 'model'
+  ) as IComboWidget
+  const durationWidget = node.widgets?.find(
+    (w) => w.name === 'duration'
+  ) as IComboWidget
+  const resolutionWidget = node.widgets?.find(
+    (w) => w.name === 'resolution'
+  ) as IComboWidget
+
+  const fallback = '$0.04-0.24/second'
+  if (!modelWidget || !durationWidget || !resolutionWidget) return fallback
+
+  const model = String(modelWidget.value).toLowerCase()
+  const resolution = String(resolutionWidget.value).toLowerCase()
+  const seconds = parseFloat(String(durationWidget.value))
+  const priceByModel: Record<string, Record<string, number>> = {
+    'ltx-2 (pro)': {
+      '1920x1080': 0.06,
+      '2560x1440': 0.12,
+      '3840x2160': 0.24
+    },
+    'ltx-2 (fast)': {
+      '1920x1080': 0.04,
+      '2560x1440': 0.08,
+      '3840x2160': 0.16
+    }
+  }
```

Comment:
Not sure if it's worthwhile yet, but using `as const` here and checking the keys could add some safety, make it less likely we ever hit the fallback.
```suggestion
  const priceByModel = {
    'ltx-2 (pro)': {
      '1920x1080': 0.06,
      '2560x1440': 0.12,
      '3840x2160': 0.24
    },
    'ltx-2 (fast)': {
      '1920x1080': 0.04,
      '2560x1440': 0.08,
      '3840x2160': 0.16
    }
  } as const
```

Reply:
I hope we can soon formalize API node pricing into a declarative schema and move the calculation logic to the Python backend.

Any idea why backport failed?